backend/src/agents/retriever_agent.py

```diff
@@ -23,170 +31,203 @@ def __init__(self) -> None:
-            docs = RetrieverAgent.general_retriever.invoke(input=query)
-
-        doc_text = ''
-        doc_srcs = []
-        for doc in docs:
-            doc_text += f'\n\n- - - - - - - - - - - - - - - \n\n{doc.page_content}'
-            if 'url' in doc.metadata:
-                doc_srcs.append(doc.metadata['url'])
-            elif 'source' in doc.metadata:
-                doc_srcs.append(doc.metadata['source'])
+        if RetrieverAgent.general_retriever is None:
+            raise ValueError('General Retriever not initialized')
 
-        return doc_text, doc_srcs
+        docs = RetrieverAgent.general_retriever.invoke(input=query)
+        return format_docs(docs)
 
     @staticmethod
     @tool
-    def retrieve_cmds(query: str) -> Tuple[str, list[str]]:
+    def retrieve_cmds(query: str) -> Tuple[str, list[str], list[str]]:
         """
-        The Command Retriever Tool is designed to provide detailed and comprehensive information on the commands and tools available in the OpenROAD project and OpenROAD-Flow-Scripts. This includes descriptions, usage guidelines, command syntax, examples, and best practices. The tool assists users by delivering clear, accurate, and relevant details to help them effectively utilize the following commands and tools within OpenROAD and OpenROAD-Flow-Scripts:
-
-            Antenna Rule Checker
-            Clock Tree Synthesis
-            Design For Testing
-            Detailed Placement
-            Detailed Routing
-            Metal Fill
-            Floorplanning
-            Global Placement
-            Global Routing
-            Graphical User Interface
-            Initialize Floorplan
-            Macro Placement
-            Hierarchical Macro Placement
-            OpenDB
-            Chip-level Connections
-            Pad
-            Partition Manager
-            Power Distribution Network
-            Pin Placement
-            IR Drop Analysis
-            Parasitics Extraction
-            Restructure
-            Gate Resizer
-            Rectilinear Steiner Tree
-            TapCell
-            Read Unified Power Format
-            Utilities
-
-        This tool aims to be an essential resource for users, providing all necessary information to maximize their productivity and efficiency when working with OpenROAD and OpenROAD-Flow-Scripts.
-        """
-        if RetrieverAgent.commands_retriever is not None:
-            docs = RetrieverAgent.commands_retriever.invoke(input=query)
+        Retrieve information on the commands and tools available in the OpenROAD project and OpenROAD-Flow-Scripts. \
+        This includes descriptions, usage guidelines, command syntax, examples, and best practices.
+        The tool assists users by delivering clear, accurate, and relevant details to help them effectively utilize the following commands and tools within OpenROAD and OpenROAD-Flow-Scripts:
+ 
+        Here's a list of all the tools available:
+        Antenna Rule Checker
```

Comment:
Do you think it would make more sense to the LLM if you do ANT: Antenna Rule Checker (ie. abbreivation inline?)